components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -194,17 +193,39 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
       // When in this state, we expect it to become running or stopping anytime soon.
       case "interrupted":
         phase = StartPhase.Running;
-        statusMessage = <p className="text-base text-gray-400">Checking On Workspace …</p>;
+        statusMessage = <p className="text-base text-gray-400">Checking workspace …</p>;
         break;
 
       // Stopping means that the workspace is currently shutting down. It could go to stopped every moment.
       case "stopping":
-        statusMessage = <p className="text-base text-gray-400">Stopping …</p>;
+        phase = StartPhase.Stopping;
+        statusMessage = <div>
+          <div className="flex space-x-3 items-center rounded-xl py-2 px-4 h-16 w-72 mt-4 bg-gray-100">
+            <div className="rounded-full w-3 h-3 text-sm bg-gitpod-kumquat">&nbsp;</div>
+            <div>
+              <p className="text-gray-700 font-semibold">{this.state.workspaceInstance.workspaceId}</p>
+              <p>{this.state.contextUrl}</p>
+            </div>
+          </div>
+        </div>;
         break;
 
       // Stopped means the workspace ended regularly because it was shut down.
       case "stopped":
-        statusMessage = <p className="text-base text-gray-400">Stopped</p>;
+        phase = StartPhase.Stopped;
+        statusMessage = <div>
+            <div className="flex space-x-3 items-center rounded-xl py-2 px-4 h-16 w-72 mt-4 bg-gray-100">
+            <div className="rounded-full w-3 h-3 text-sm bg-gray-300">&nbsp;</div>
+            <div>
+              <p className="text-gray-700 font-semibold">{this.state.workspaceInstance.workspaceId}</p>
+              <p>{this.state.contextUrl}</p>
+            </div>
+          </div>
+          <div className="mt-10 flex space-x-2">
+            <button className="px-4 py-2 text-gray-500 bg-white font-semibold border-gray-500" onClick={() => this.redirectTo(gitpodHostUrl.asDashboard().toString())}>Go to Dashboard</button>
+            <button className="px-4 py-2 text-gray-50 bg-green-600 font-semibold border-green-800" onClick={() => this.redirectTo(gitpodHostUrl.asStart(this.state.workspaceInstance?.workspaceId).toString())}>Open Workspace</button>
```

Comment:
**issue**: Both of these buttons lack hover state. Maybe we could add this on the button component in [`index.css`](https://github.com/gitpod-io/gitpod/blob/dashboard-v2/components/dashboard/src/index.css#L31-L33).

Reply:
Moved to https://github.com/gitpod-io/gitpod/issues/3507.